install.sh

```diff
@@ -116,11 +116,6 @@ install_packages( )
     library_dependencies_ubuntu+=( "cuda" )
     library_dependencies_centos+=( "" ) # how to install cuda on centos?
     library_dependencies_fedora+=( "" ) # how to install cuda on fedora?
-  elif [[ "${build_hip_clang}" == false ]]; then
-    library_dependencies_ubuntu+=( "rocm-dev" "rocblas" )
-    library_dependencies_centos+=( "rocm-dev" "rocblas" )
-    library_dependencies_fedora+=( "rocm-dev" "rocblas" )
-    library_dependencies_sles+=( "rocm-dev" "rocblas" )
   fi
```

Comment:
Could you please add -z flag to specify custon_rocm_dev like rocBLAS PR#1041. Also add custom flag form custom_rocblas_dev. https://github.com/ROCmSoftwarePlatform/rocBLAS/pull/1041/files

Reply:
Made changes as suggested

Reply:
Made changes as suggested